### ISSUE TITLE
Bug in VS 17.8.3 - MAUI - for windows .net8.0

### ISSUE THREAD

**Issue by vsfeedback** [dotnet/maui#20272]
_This issue has been moved from [a ticket on Developer Community](https://developercommunity.visualstudio.com/t/Bug-in-VS-1783---MAUI---for-windows-n/10555112)._

---
[severity:I'm unable to use this version]
My windows PC:

Versjon	Windows 11 Pro
Versjon	22H2
Installert den	‎21.‎09.‎2022
Operativsystembygg	22621.2861
Serienummer	PF2YNAQH
Opplevelse	Windows Feature Experience Pack 1000.22681.1000.0

target windows framework 10.0.19041.0

I have a self coded statusbar with icons (png images) and text (label) below.
This worked fine with .net7.0.

Only moved to .net8.0 and this bug appeared.

Statusbar can be any number of "icon-cells" (mostly btw 2 and 8) in a horizontal stacklayout.

Each "cell" in statusbar is a vertical StackLayout with two children:  png-image and text (label).

App functionality can enable/disable cell. That works fine.... (When disabled, no press is registered)

When the icon-cell (parent) is enabled, I use (for the two children) a white icon and set text color to white (label. TextColor = Color.FromArgb("ffffff") )
This works perfect.

But, when the icon-cell (parent) is disabled, I use (for the two children) a gray icon and set text color to gray (label. TextColor = Color.FromArgb("777777") )
The: the icon is changes to gray, but the label-text becoms invisible.

I have tried to not disable the parent-cell, then the label-text is displayed correctly.

To summarize bug for .net8.0:  when disabling a stacklayout(vertical), its children as labels becomes invisible

---
#### Feedback Bot on 1/8/2024, 06:01 PM: 

(private comment, text removed)

---
### Original Solutions
(no solutions)

**Comment by Mek7** [comment]
I think I have the same problem. I am creating a Label with a TextColor set to red, both in code-behind (not xaml), but in runtime the text color is gray instead of red.
The exact same code worked OK in .net 7.0 (the label was red).
Looking forward to a fix!

**Comment by ninachen03** [comment]
Verified this issue with Visual Studio 17.10 Preview 7 (8.0.21 & 8.0.20). Can not repro it.
[MauiApp39.zip](https://github.com/dotnet/maui/files/15347079/MauiApp39.zip)

**Comment by Mek7** [comment]
I'd also like to test this but I can't do it for net7.0 for comparison. In net8.0 my problem is still present.
After I installed VS Preview, I successfully compiled and ran my MAUI project in android emulator, compiled for net8.0 - font color is still incorrectly gray there, instead of red.
Then I made a copy of the solution folder and changed target frameworks to net7.0-android, deleted .vs folder and bin and obj folders. Now I can't even compile, because:
`The targeting pack Microsoft.Android is not installed. Please restore and try again.	Quad (net7.0-android)	C:\Program Files\dotnet\sdk\8.0.300-preview.24203.14\Sdks\Microsoft.NET.Sdk\targets\Microsoft.NET.Sdk.FrameworkReferenceResolution.targets`

Nevertheless, I am attaching my solution in a zip file.
[Quad-net8.zip](https://github.com/dotnet/maui/files/15369500/Quad-net8.zip)
Look into method MakeDividerLabel() where TextColor property of Label is set to red, but it is gray instead when run in emulator (and also in real Android phone). The Label is the "arrows" next to the red X in the following screenshot, they should also be red but they are gray and so barely visible.
![net8](https://github.com/dotnet/maui/assets/12199478/b1ba6233-c24e-450a-93c4-3ec4089682e5)
I hope this still can be fixed...?

**Comment by Mek7** [comment]
You closed this issue, that's nice of you... that you don't even read what people write.
